http-server-netty/src/main/java/io/micronaut/http/server/netty/converters/NettyConverters.java

```diff
@@ -347,13 +341,6 @@ public void register(ConversionService<?> conversionService) {
         };
     }
 
-    /**
-     * @return A converter that returns bytebufs to objects
-     */
-    protected TypeConverter<ByteBuf, Object> byteBufToObjectConverter() {
-        return (object, targetType, context) -> conversionService.convert(object.toString(context.getCharset()), targetType, context);
-    }
-
```

Comment:
Is this not a breaking change?

Reply:
I couldn't find any uses of this (which does a strange `toString` conversion that interferes with other more useful conversions). No tests broke, and no one seemed to think this was a problem when it was brought up during last engineering meeting.

Reply:
@graemerocher The presence of the converter prevents proper conversion of ByteBuf to container types. In general I don't think it makes sense because if we don't know the target type we shouldn't assume that a string based conversion will succeed or will be desirable